assets/src/scss/_dependencies.scss

```diff
@@ -29,10 +29,14 @@
  *
  */
 
-@include card();
```

Comment:
Need to add this back in.

Reply:
It's not an optional component anymore – it's included in the base

Reply:
I remember talking about this now 😸